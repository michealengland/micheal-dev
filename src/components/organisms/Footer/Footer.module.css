.footer {
  background-color: #020041;
  color: #ffffff;
  position: relative;
  width: 100%;

  & .inner {
    /* marginLeft: `auto`,
    marginRight: `auto`,
    maxWidth: isBlog( blogPath ) ? rhythm( 30 ) : '',
    padding: isBlog( blogPath ) ? `2em` : '', */
  }
}
