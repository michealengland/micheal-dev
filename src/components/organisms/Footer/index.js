import React from 'react'

export default function Footer() {
  return (
    <footer>
      <p>Â© {new Date().getFullYear()} Mike England, Built with <a href="https://www.gatsbyjs.org">Gatsby</a></p>
    </footer>
  )
}
