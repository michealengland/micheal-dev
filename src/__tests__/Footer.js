import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '@components/organisms/Footer'
describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Footer />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
