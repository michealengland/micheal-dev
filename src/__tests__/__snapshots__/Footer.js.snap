// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="inner"
  >
    <p>
      Â© 
      2021
       Mike England | View on
       
      <a
        href="https://github.com/michealengland/micheal-dev"
      >
        Github
      </a>
    </p>
  </div>
</footer>
`;
